### ISSUE TITLE
Make section-info-option accessible

### ISSUE THREAD

**Issue by Mardak** [mozilla/activity-stream#2997]
From https://github.com/mozilla/activity-stream/issues/2874#issuecomment-318425638

section-info-option is not accessible (for example screen/keyboard reader user will never be able to send feedback):
- it has to be in tab index
- it has also no semantic information (could be role="note")
- sr-only could be eliminated in favour title or aria-label
- it should have aria-haspopup, aria-controls and aria-expanded attributes (I would expand it as soon as it's focused)
- you can probably use aria-describedby attribute here to point at the info-option header and body
- info-option-header should have header semantics.

**Comment by dmose** [comment]
Things turned out to be a bit more complicated than expected.  After talking to Yura, it seems clear that the right thing is to wave this off until 57 rather than trying to jam it in at the last minute tonight.
